packages/viz/src/lib/sources/Source.ts

```diff
@@ -8,6 +8,7 @@ export interface LayerProps {
   geometryType: string;
 }
 
+export type bins = 'quantiles' | 'stdev' | 'equal';
```

Comment:
I'd name this type as `Classifier` or similar to be more clarifying

```javascript
export type Classifier = 'quantiles' | 'stdev' | 'equal';
```

Reply:
Agree

Reply:
Done